RoT/Protocol/RFCs/0002-TCG_Compliant_Logs.md

```diff
@@ -0,0 +1,44 @@
+* Name: TCG Comliant Measurement Logs
+* Date: 2021-05-07
+* Pull Request: [#17](https://github.com/opencomputeproject/Security/pull/17)
+
+# Objective
+
+The Cerberus Challenge Protocol provides several mechanisms for accessing measurements
+and measured data.  This includes the Attestation Log, which generates a list of
+individual measurements.  While this structure is inspired by TCG-style logging, it
+is not compatible with parsers of that format.  It is advantagous from an attestation
+perspective to have the ability to get fully compliant TCG logs from the device.
+This can provide some uniformity with TPM attestation flows, especialy when paired
+with the Unseal command.
+
+# Proposal
+
+Update the Get Log command to support an additional Log Type that would provide a
+TCG log generated by the device.  The Get Log Info command would not be updated at
+this time, as the log sizes, especially for the TCG Log, are not that relevant in
+practice.
+
+In the same way that the Get Log command is an optional command, it is not required
+that every device support generating TCG-compliant logs.
+
+# Specification Changelist
+
+Add Log Type '4' to the table of supported types for TCG logs.  The descriptive
+text for the Get Log command would also need to be updated to reflect the addition
+of the new type.
```

Comment:
I think it might be useful to link to the relevant TGC spec that lays out the structure?